rio/package.yaml

```diff
@@ -7,6 +7,7 @@ author: Michael Snoyman
 maintainer: michael@snoyman.com
 github: commercialhaskell/rio
 category: Control
+
```

Comment:
Can you also include a minor version bump here, a changelog entry, and a `@since` comment on the new module's Haddocks?